contracts/mirror/src/contract.rs

```diff
@@ -56,33 +54,31 @@ pub fn query(deps: Deps<NeutronQuery>, _env: Env, msg: QueryMsg) -> ContractResu
     match msg {
         QueryMsg::Ownership {} => Ok(to_json_binary(&cw_ownable::get_ownership(deps.storage)?)?),
         QueryMsg::Config {} => Ok(to_json_binary(&CONFIG.load(deps.storage)?)?),
-        QueryMsg::One { id } => query_one(deps, id),
-        QueryMsg::All { start_after, limit } => query_all(deps, start_after, limit),
+        QueryMsg::FailedReceiver { receiver } => query_failed_receiver(deps, receiver),
+        QueryMsg::AllFailed {} => query_all_failed(deps),
     }
 }
 
-pub fn query_one(deps: Deps<NeutronQuery>, id: u64) -> ContractResult<Binary> {
-    let bond = BONDS.load(deps.storage, id)?;
-    to_json_binary(&bond).map_err(ContractError::Std)
+fn query_all_failed(deps: Deps<NeutronQuery>) -> ContractResult<Binary> {
+    let failed_transfers: Vec<(String, Vec<Coin>)> = FAILED_TRANSFERS
+        .range(deps.storage, None, None, cosmwasm_std::Order::Ascending)
+        .map(|pair| {
+            let (receiver, funds) = pair.unwrap();
+            (receiver, funds)
```

Comment:
You can simply unwrap here
```suggestion
            pair.unwrap()
```